### PR TITLE
Add the zsh snap

### PR DESCRIPTION
**Pull request by elopio**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ubuntu/snappy-playpen/pull/238%23issuecomment-247203968%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/238%23issuecomment-247203968%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40elopio%20nice%20work%21%5Cr%5Cn%5Cr%5CnI%20haven%27t%20tested%20it%2C%20but%20I%20looked%20it%20over%20and%20it%20seems%20to%20be%20%3A%2B1%3A%20except%20for%20the%20comments%20I%20left.%22%2C%20%22created_at%22%3A%20%222016-09-15T01%3A05%3A52Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3689821%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tsimonq2%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/ubuntu/snappy-playpen/pull/238#issuecomment-247203968'>General Comment</a></b>
- <a href='https://github.com/tsimonq2'></a> @elopio nice work!
I haven't tested it, but I looked it over and it seems to be :+1: except for the comments I left.

<a href='https://www.codereviewhub.com/ubuntu/snappy-playpen/pull/238?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ubuntu/snappy-playpen/pull/238?approve=1'></a>&nbsp;<a href='https://github.com/ubuntu/snappy-playpen/pull/238'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by didrocks** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/ubuntu/snappy-playpen/pull/238'>Refresh</a>.<a href='#crh-update-none'></a>

File: zsh/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Z shell snap
+
+This project creates a devmode working snap of the Z shell.
```
Comment (tsimonq2) [inline zsh/README.md]: I would reword this.

File: zsh/snapcraft.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: zsh
+version: master
+summary: a shell designed for interactive use, and a powerful scripting language
+description: |
+  The Z-Shell, `zsh' for short, is a command interpreter for UNIX systems. zsh
```
Comment (tsimonq2) [inline zsh/snapcraft.yaml]: mismatching single quotes

File: zsh/snapcraft.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+apps:
+  zsh:
+    command: zsh
+
+parts:
+  zsh:
+    source: git://git.code.sf.net/p/zsh/code
+    plugin: autotools
+    install-via: prefix
+    build-packages: [yodl, libncursesw5-dev]
```
Comment (tsimonq2) [inline zsh/snapcraft.yaml]: I would suggest reversing the two.

wrap-and-sort is nice and makes me think of these things. We need one for snaps. :D

**Comment by tsimonq2** [comment]
@elopio nice work!

I haven't tested it, but I looked it over and it seems to be :+1: except for the comments I left.

**Comment by elopio** [comment]
Thanks for the review. I pushed the fixes.

**Comment by dholbach** [comment]
Nice work! Merging!
